rpc/amqp-rpc.go

```diff
@@ -171,7 +173,7 @@ type AmqpRPCServer struct {
 
 // NewAmqpRPCServer creates a new RPC server for the given queue and will begin
 // consuming requests from the queue. To start the server you must call Start().
-func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer)) (*AmqpRPCServer, error) {
+func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer), durable, autoAck bool) (*AmqpRPCServer, error) {
```

Comment:
Which is why this just needs to be a separate PR. It's a big change. My preview: the end-to-end argument says this doesn't matter and we're already screwed by using distributed queues in-between our processes.

I question why we'd need to make the queue durable if we're still creating a new goroutine to pass the RPC off instead of blocking on the response.